doc/classes/Curve2D.xml

```diff
@@ -107,11 +107,12 @@
 			<param index="0" name="offset" type="float" default="0.0" />
 			<param index="1" name="cubic" type="bool" default="false" />
 			<description>
-				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve. Returns empty transform if length of the curve is [code]0[/code].
+				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve, with [member Transform2D.origin] as the point position, [member Transform2D.x] as the sideways vector, and [member Transform2D.y] as the forward vector. Returns empty transform if length of the curve is [code]0[/code].
+				[b]Note:[/b] To use this to transform nodes you need to do:
 				[codeblock]
-				var transform = curve.sample_baked_with_rotation(offset)
-				position = transform.get_origin()
-				rotation = transform.get_rotation()
+				var baked = curve.sample_baked_with_rotation(offset)
```

Comment:
Swapped here to not suggest using a colliding variable called `transform`, just for clean code if used in `Node2D`

Reply:
Maybe you can also add something like: `or you can use a scaled copy of the Transform for your movement directly:`
```go
var transform = curve.sample_baked_with_rotation(offset)
target.transform = transform.scaled_local(Vector2(1.0, -1.0))
```

Reply:
Thank you, will take a look at expanding the suggestion